### PR TITLE
named-imports - export missing components

### PR DESCRIPTION
**Pull request by mykas**
This PR is fixing missing named exports after testing the codemode of named-imports on events project.

### CODE REVIEW

File: src/index.d.ts [hunk 1]
```diff
@@ -1,7 +1,14 @@
 import * as React from 'react';
 
-export { IconElement, InjectedFocusableProps, Omit } from './common';
 
+import { default as PopoverMenuBeta } from './beta';
+
+// NEXT section to rexport as named import from NEXT namespace
+export const NEXT = () => ({});
```
Comment (shlomitc) [inline src/index.d.ts]: Maybe instead of adding an additional entry, just name it as `PopoverMenuBeta`

Comment (simast) [reply src/index.d.ts]: alternatively maybe we should not expose beta/next components form the main index file but keep it in /beta/index.js? The users would then use (and our codemod could convert/handle this as well to):

```js
import { PopoverMenu } from 'wix-style-react/beta'
```

Note that we already have `index.js` in `/beta`.

File: src/index.d.ts [hunk 2]
```diff
@@ -1,7 +1,14 @@
...
+NEXT.PopoverMenu = PopoverMenuBeta;
+
+
+export { IconElement, InjectedFocusableProps, Omit } from './common';
```
Comment (simast) [inline src/index.d.ts]: is this a public api? have you found where users are using types from `/common`?

File: src/index.d.ts (hunk 1 above)
Comment (mykas) [reply src/index.d.ts]: Well, I think it is easier to just go with `PopoverMenuNext` instead. I am not sure how to handle it there properly.

File: src/index.d.ts (hunk 2 above)
Comment (mykas) [reply src/index.d.ts]: it is public api, but we are not promoting yet  named imports so we might delete those just before users start using it? what do you think @shlomitc

Comment (simast) [reply src/index.d.ts]: my bad, I see the types were already exposed before your changes

**Pull request merged**